docs/getting-started/installation/linux.md

```diff
@@ -4,92 +4,91 @@ sidebar_position: 3
+- `libz` and `libssl` (unless you know your distribution uses something else, this means OpenSSL) development headers
+- `curl`, an HTTP (among others) client
+- Various transitive dependencies of [PostgreSQL and its compilation](https://github.com/smashedtoatoms/asdf-postgres#dependencies) not already listed above: 
+    * `libreadline-dev`
+    * `libcurl4-openssl-dev`
+    * `uuid-dev`
+    * `icu-devtools`
 
-There are more than one ways to setup PostgreSQL. For additional configuration,
-check out our [PostgreSQL setup guide](postgresql.md) or the official
-[PostgreSQL](https://www.postgresql.org/) site for further information.
+```sh
+sudo apt update
+sudo apt install -y imagemagick curl build-essential pkg-config libssl-dev libz-dev libreadline-dev libcurl4-openssl-dev uuid-dev icu-devtools
+```
 
-### ImageMagick
+## Installing Version-Locked Tool Dependencies
 
-Forem uses [ImageMagick](https://imagemagick.org/) to manipulate images on
-upload.
+> This is the part where, if you've opted out of `rtx`, you'll need to round up tools on your own. For Ruby, consider setting up `rbenv` using their [installation guide](https://github.com/rbenv/rbenv#installation) and then installing our current Ruby version target with `rbenv install $(cat .ruby-version)`. For NodeJS, consider `nvm` and install our Node version target with `nvm install`. You're on your own for installing [Yarn v1](https://yarnpkg.com/en/docs/install), [PostgreSQL v13](https://www.postgresql.org/) (perhaps via [Postgres.app](https://postgresapp.com/), also see our [auxiliary documentation](postgresql.md)), and [Redis](https://redis.io/docs/getting-started/installation/install-redis-on-mac-os/).
 
-Please refer to ImageMagick's
-[instructions](https://imagemagick.org/script/download.php) on how to install
-it.
+Use `rtx install` - this will pull the exact versions we recommend for the tools listed in the block above, which helps to prevent "version drift" - where someone writes code targeting Version A of some tool, but it breaks when run against Version B of that same tool.
 
-### Redis
+If prompted, respond `y` to the prompts about installing plugins to handle Yarn, PostgreSQL, and Redis.
 
-Forem requires Redis version 6.0 or higher.
+**This step will probably take several minutes**: it's building most of these dependencies from source code. Errors at this point are unlikely, but if they pop up, should be expected to be quite varied: searching for error messages will often lead to helpful solutions, as few to none of these errors are likely to have never been seen (and triaged) before.
 
-We recommend following Digital Ocean's extensive guides
-[How to Install and Secure Redis](https://www.digitalocean.com/community/tutorial_collections/how-to-install-and-secure-redis)
-to setup Redis.
+## Last Bits Of Setup
 
-## Installing Forem
+1. Start the caching server with `redis-server --daemonize yes` (do this every time you work on Forem!)
```

Comment:
should we include additional instruction on how they can have it automatically start on launch with
```
sudo systemctl enable postgresql
sudo systemctl enable redis-server
```